tests/pc/test_po_cleanup.py

```diff
@@ -62,3 +60,45 @@ def test_po_cleanup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_as
     if not wait_until(10, 1, 0, check_kernel_po_interface_cleaned, duthost, enum_asic_index):
         fail_msg = "PortChannel interface still exists in kernel"
         pytest.fail(fail_msg)
+    # Restore swss service.
+    duthost.asic_instance(enum_asic_index).start_service("swss")
+
+def test_po_cleanup_after_reload(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
+    """
+    test port channel are cleaned up correctly after config reload, with system under stress.
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    host_facts = duthost.setup()['ansible_facts']
+
+    # Get the cpu information.
+    if host_facts.has_key("ansible_processor_vcpus"):
+        host_vcpus = int(host_facts['ansible_processor_vcpus'])
+    else:
+        res = duthost.shell("nproc")
+        host_vcpus = int(res['stdout'])
+
+    logging.info("found {} cpu on the dut".format(host_vcpus))
+
+    # Get portchannel facts and interfaces.
+    lag_facts = duthost.lag_facts(host = duthost.hostname)['ansible_facts']['lag_facts']
+    port_channel_intfs = lag_facts['names'].keys()
+
+    # Add start marker to the DUT syslog
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix='port_channel_cleanup')
+    loganalyzer.expect_regex = []
+    for pc in port_channel_intfs:
+        loganalyzer.expect_regex.append(LOG_EXPECT_PO_CLEANUP_RE.format(pc))
+
+    try:
+        # Make CPU high
+        for i in range(host_vcpus):
+            duthost.shell("nohup yes > /dev/null 2>&1 & sleep 1")
+
+        with loganalyzer:
+            logging.info("Reloading config..")
+            config_reload(duthost)
```

Comment:
I was thinking if all the CPUs are 100% occupied, can we lose access to the device? How can we stop all running `yes` if that happens?

Reply:
We have the "sleep 1" which **prevents** the CPU from going 100% CPU and not responsive. I have verified it is ok.